installer/templates/phx_web/components/core_components.ex

```diff
@@ -295,7 +295,7 @@ defmodule <%= @web_namespace %>.CoreComponents do
   def input(%{field: %Phoenix.HTML.FormField{} = field} = assigns) do
     assigns
     |> assign(field: nil, id: assigns.id || field.id)
-    |> assign(:errors, Enum.map(field.errors, &translate_error(&1)))
+    |> assign(:errors, Enum.map(Phoenix.Component.used_errors(field), &translate_error(&1)))
```

Comment:
In my head we'd include `used_errors` directly in CoreComponent. I don't think we have a precedent for helper input functions in Phoenix.Component, do we?